src/EventProducerService.cpp

```diff
@@ -213,26 +221,54 @@ void EventProducerService::handleProdSvcMessage(const VlcbMessage *msg)
 {
   unsigned int opc = msg->data[0];
   unsigned int nn = Configuration::getTwoBytes(&msg->data[1]);
-  // DEBUG_SERIAL << ">VLCBProdSvc handling message op=" << _HEX(opc) << " nn=" << nn << " en" << en << endl;
-
-  switch (opc) 
-  {    
-
-    case OPC_AREQ:
-      // AREQ message - request for node state, only producer nodes
-
-      if ((nn == module_config->nodeNum) && (eventhandler != nullptr)) 
+  unsigned int en = Configuration::getTwoBytes(&msg->data[3]);
+  
+  if (requesteventhandler != nullptr)
+  {
+    switch (opc)
+    {
+      case OPC_ASRQ:
```

Comment:
Reading op-code spec on ASRQ: If NN is not zero and NN != this node number then ignore this request.

Reply:
You're quite correct, of course.  Now done.